src/commands/mod.rs

```diff
@@ -1872,11 +1871,12 @@ fn cmd_duplicate(
 
     let mut tx = workspace_command
         .start_transaction(&format!("duplicating {} commit(s)", to_duplicate.len()));
-    let index = tx.base_repo().readonly_index().clone();
-    let store = tx.base_repo().store().clone();
+    let base_repo = tx.base_repo().clone();
+    let store = base_repo.store().clone();
```

Comment:
nit: no more need to `.clone()` the store

Reply:
Fixed.